### PR TITLE
github: add issue and pull request templates

### PR DESCRIPTION
**Pull request by XadillaX**
Refs: https://github.com/nodejs/node/pull/5291

##### Checklist

- [x] documentation is changed or added
- [x] commit message follows Node.js [commit guidelines](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#commit-message-guidelines)

### CODE REVIEW

**Review by cleverboy32 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!--
+Thank you for reporting an issue.
+
+This issue tracker is for bugs and issues found within Akyuu.js core.
+
+
```
Comment (xiaoqiangsdl) [inline .github/ISSUE_TEMPLATE.md:6]: 多了空行？

Comment (XadillaX) [reply .github/ISSUE_TEMPLATE.md:6]: 这是照着 Node.js 来的。

**Review by xiaoqiangsdl — APPROVED** [review]
LGTM

**Comment by sallen450** [comment]
Landed in https://github.com/akyuujs/akyuu/commit/8559ae4ae2dc6ea80e117238401a8d7b832ec1fd

**Pull request closed without merging**